### PR TITLE
feat: accept and support preauth at any time

### PR DESCRIPTION
**Pull request by merlin-northern**
test: https://github.com/mendersoftware/integration/pull/2576

Changelog: Title
Ticket: MEN-6961

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

**Comment by merlin-northern** [comment]
@kjaskiewiczz I am looking at a cool way to test it as a whole -- backend integration test I will add probably. (we already have some)

**Comment by merlin-northern** [comment]
> @kjaskiewiczz I am looking at a cool way to test it as a whole -- backend integration test I will add probably. (we already have some). so far tested manually. at least one of the versions :) of this code.

and do not worry -- I will test it manually some more.

**Comment by merlin-northern** [comment]
passed pipeline with https://github.com/mendersoftware/integration/pull/2576: https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/6405781880

File: docs/management_api.yml [hunk 1]
```diff
@@ -559,6 +559,11 @@ definitions:
   PreAuthSet:
     type: object
     properties:
+      force:
+        type: boolean
+        description: >
+          Indicates that despite the device having accepted authsets the preauthorization
```
Comment (kjaskiewiczz) [inline docs/management_api.yml:565]: Without this flag, device with any authset (accepted, pending, rejected) will prevent from preauthorization.

**Review by kjaskiewiczz — APPROVED** [review]
looks good

File: docs/management_api.yml (hunk 1 above)
Comment (merlin-northern) [reply docs/management_api.yml]: fixed.

**Pull request merged**